src/Events/Category_Colors/Event_Category_Meta.php

```diff
@@ -106,9 +106,9 @@ public function set_term( int $term_id ): self {
 	 *
 	 * @return mixed The meta value, or an array of all metadata if no key is provided.
 	 */
-	public function get( ?string $key = null ) {
+	public function get( ?string $key = '' ) {
```

Comment:
default to null - an allowed value that triggers "get all" anyway. why the empty string?